govcd/vm.go

```diff
@@ -85,6 +86,35 @@ func (vm *VM) GetNetworkConnectionSection() (*types.NetworkConnectionSection, er
 	return networkConnectionSection, err
 }
 
+// UpdateNetworkConnectionSection applies network configuration of types.NetworkConnectionSection for the VM
+// Runs synchronously, VM is ready for another operation after this function returns.
+func (vm *VM) UpdateNetworkConnectionSection(networks *types.NetworkConnectionSection) error {
```

Comment:
Usually Giuseppe request to have functions which returns tasks. @dataclouder

Reply:
No issues with this code from my side